docs/blockchain/contracts/nft-contract/nft-actions/rmfctofr.a.experimental.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'rmfctofr.a'
+order: 42
+
+---
+
+# rmfctofr.a
+
+Cancel the buy offer made on the Uniq factory.
```

Comment:
Should be 
Cancel the offer made on ...

Reply:
updated